docs/code-standards.md

```diff
@@ -0,0 +1,42 @@
+# Code standards
+
+General code standards.
+
+- [Enums and constants](#enums-and-constants)
+- [Functions](#functions)
+
+## Enums and constants
+
+Following the naming conventions for [JavaScript](https://google.github.io/styleguide/javascriptguide.xml?showone=Naming#Naming) / [TypeScript](https://www.typescriptlang.org/docs/handbook/enums.html):
+
+- Constant values should be upper snake case (`CONSTANT_VALUES_LIKE_THIS`)
+- Enums should be named in singular PascalCase with values also in PascalCase
+
+```ts
+// Example enum
+enum Direction {
+  Up,
```

Comment:
Just wondering if this should be
```
enum Direction {
  Up: "Up"
}
```
since this would just default fo 0,1,2,3

Reply:
I pulled that example from the TS docs, but it just depends on what you want your enum values to be. Technically there's nothing wrong with it being int values since it's the default, but you're right in that in most cases we would probably actually assign string values.

I don't mind amending it, but this was more about how to format an enum rather than how to use one (since there are official docs for that).

Reply:
Fair enough - more of a consistency thing than anything else